GameFighter/Models/Vampire.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace GameFighter.Models
+{
+    public class Vampire : Warrior
+    {
+        public int Vampirism { get; protected set; }
+
+        public Vampire()
+        {
+            Health = 40;
+            Attack = 4;
+            Vampirism = 50;
+        }
+
+        public void Healing(int actualAttack)
```

Comment:
This logic shouldn't be in the separate method. Description says that Vampire has this additional healing ability when he attacks